src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+@import 'utils/variables';
+@import 'utils/mixin';
+@import 'blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  .imac {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    .name {
+      font-family: Roboto, sans-serif;
+
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: #060b35;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    .code {
+      font-family: Roboto, sans-serif;
+
+      margin-block: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: #616070;
+    }
+  }
+
+  .rating {
+    @include spacebetweenblock;
+
+    align-items: center;
+    width: 100%;
+    margin-bottom: 24px;
+
+    &__reveiws {
+      font-family: Roboto, sans-serif;
+      margin-block: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    @include spacebetweenblock;
+
+    width: 100%;
+    margin-bottom: 16px;
+
+    .price {
+      font-family: Roboto, sans-serif;
+      margin-block: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    .amount {
+      font-family: Roboto, sans-serif;
+      margin-block: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #000;
+    }
+  }
+
+  .button {
```

Comment:
If it is a separate block move it to a separate file. Otherwise make it part of the card BEM block